include/cutlass/gemm/collective/builders/xe_mma_builder.inl

```diff
@@ -68,9 +68,10 @@ struct CollectiveBuilder<
   KernelScheduleType,
   cute::enable_if_t<
     (cute::is_same_v<KernelScheduleType, KernelPVC> ||
-     cute::is_same_v<KernelScheduleType, KernelScheduleAuto>) &&  
-    cute::is_same_v<GmemLayoutATag, cutlass::layout::RowMajor> && // Different struct specialization because this will change copy atoms
-    cute::is_same_v<GmemLayoutBTag, cutlass::layout::RowMajor>
+      cute::is_same_v<KernelScheduleType, KernelScheduleAuto>) &&
+    cute::is_same_v<ElementA, ElementB> &&
+    cute::is_any_of_v<ElementA, bfloat16_t, half_t> &&
+    cute::is_any_of_v<ElementB, bfloat16_t, half_t>
```

Comment:
this is a repetition of the static_assert. You should probably only have one or the other.

Reply:
removed asserts